Log config index (#4648)
DataDog/documentation: content/en/api/logs_indexes/get_indexes_order.md

---
title: Get Indexes order
type: apicontent
order: 23.4
external_redirect: /api/#get-indexes-order
---

## Get Indexes Order

> The index order endpoints are for controlling the _order_ in which an organisation's indexes are processed. You'll use this endpoint only if you have more than one index for logs in your organisation. Note that this index cannot be used to add or delete indexes.

Reviewer: ```suggestion
Use the index order endpoints to control the processing _order_ of an organization's indexes. Use this endpoint only if you have more than one index for logs in your organization. Note that this index cannot be used to add or delete indexes.
```

---
title: Get Indexes order
type: apicontent
order: 23.4
external_redirect: /api/#get-indexes-order
---

## Get Indexes Order

<div class="alert alert-warning">

> This endpoint is in public beta. Shall you have any feedback <a href="/help">Contact Datadog support</a>.

Reviewer: ```suggestion
This endpoint is in public beta. If you have any feedback, <a href="/help">contact Datadog support</a>.
```
